.gitignore

```diff
@@ -5,6 +5,7 @@
 gae_runtime/*
 third_party/*
 backend_prod_files/*
+core/templates/prod/*
```

Comment:
NB: this was removed in a recent PR, but should be reinstated, since some folks (like me) have local data in these dirs and don't want to commit it accidentally.